package.json

```diff
@@ -25,14 +25,14 @@
     "nunjucks": "^3.0.0"
   },
   "devDependencies": {
-    "babel-eslint": "^4.1.3",
+    "babel-eslint": "^7.2.3",
     "expect.js": "^0.3.1",
     "grunt": "^0.4.5",
-    "grunt-jscs": "^2.8.0",
-    "standard": "^5.3.1"
+    "snazzy": "^7.0.0",
+    "standard": "^10.0.2"
   },
   "scripts": {
-    "test": "node_modules/standard/bin/cmd.js && grunt test && node tests/all.js"
+    "test": "standard | snazzy && grunt test && node tests/all.js"
```

Comment:
Yeap, it should work. Newer versions of Node doing proper binding if you invoke it through `npm test`  — they will automatically resolve `standard` to `node_modules/standard/bin/cmd.js`. Note, that it will work only within `npm test`, if you want to invoke `standard` manually from terminal, you will need to install it globally or use that long path to node_module itself.

To be honest, I do not remember from which version of Node this works. I guess from 4 or something like this?

Just to confirm that it works, see https://vgy.me/ELh5tz.png

I have globally installed only `grunt-cli`, while `standard` invoked from local `node_modules`.